Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivRsaPublicKey.cs

```diff
@@ -66,13 +66,17 @@ public sealed class PivRsaPublicKey : PivPublicKey
         private const int ExponentTag = 0x82;
         private const int Rsa1024BlockSize = 128;
         private const int Rsa2048BlockSize = 256;
+        private const int Rsa3072BlockSize = 384;
+        private const int Rsa4096BlockSize = 512;
         private const int SliceIndex1024 = 4;
         private const int SliceIndex2048 = 5;
+        private const int SliceIndex3072 = 6;
```

Comment:
Looking for feedback on this

Reply:
Will simplify in future PR

Reply:
Have these new values been checked to make sure they are correct?

Reply:
We checked them now and it looks like they are not, including the `sliceIndex2048` which should also be 4 if packed into a 2048 byte TLV encoded array.